### PR TITLE
Update ISA in tests for D116686

### PR DESCRIPTION
**Pull request by jayfoad**
This upstream LLVM change affects codegen of s_cselect_b32 and
s_cselect_b64 instructions:

D116686 "Revert D109159 : Revert "[amdgpu] Enable selection of `s_cselect_b64`.""

### CODE REVIEW

**Comment by jayfoad** [comment]
This needs to be coordinated with a merge from upstream llvm.

**Review by s-perron — APPROVED** [review]

**Review by amdrexu — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit b0e6aac56adf9c8e50ea6bccdde2607e88e59d72
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 0a0a4ae4ab062d31fcd68863c35952967a8060ee</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev c41a825d1ea17c108e8fb123f082357f849e7823</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c20ba7614ecdc78b3d99601423783100835200c1</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/218633)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 33955/57048 (59.5%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23093/57048 (40.5%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 30946/47489 (65.2%)</li>
<li><details><summary>Failed: 1/47489 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.subgroups.clustered.graphics.subgroupclusteredmax_u16vec4
Stack trace: Script:
Crash

```
</p></details></li>

<li>Not Supported: 16541/47489 (34.8%)</li><li>Warnings: 1/47489 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 34085/57048 (59.7%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 22963/57048 (40.3%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35882/57048 (62.9%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 21166/57048 (37.1%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by amdvlk-admin** [comment]
### Test summary for commit 6ec361c90f740c13e9a7a987e542d7f07f1ffb2a
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 0a0a4ae4ab062d31fcd68863c35952967a8060ee</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev c41a825d1ea17c108e8fb123f082357f849e7823</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c20ba7614ecdc78b3d99601423783100835200c1</li>
</ul></details>

<details><summary>CTS tests (Failed: 1/218687)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 33955/57048 (59.5%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 23093/57048 (40.5%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 30989/47543 (65.2%)</li>
<li><details><summary>Failed: 1/47543 (0.0%)</summary><p>


Failures:
```
FAILURE: dEQP-VK.subgroups.clustered.compute.subgroupclusteredor_i16vec4
Stack trace: Script:
Crash

```
</p></details></li>

<li>Not Supported: 16552/47543 (34.8%)</li><li>Warnings: 1/47543 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 34085/57048 (59.7%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 22963/57048 (40.3%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35882/57048 (62.9%)</li>
<li>Failed: 0/57048 (0.0%)</li>
<li>Not Supported: 21166/57048 (37.1%)</li><li>Warnings: 0/57048 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by jayfoad** [comment]
retest this please

**Comment by amdvlk-admin** [comment]
### Test summary for commit 6ec361c90f740c13e9a7a987e542d7f07f1ffb2a
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev b638e90ca4e6e5a6fc4f00029d62f8e064aa18eb</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev 89a4cf115b9a2972ff00e2fe4959f4f1c8e7175b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c20ba7614ecdc78b3d99601423783100835200c1</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/199857)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 37645/66619 (56.5%)</li>
<li>Failed: 0/66619 (0.0%)</li>
<li>Not Supported: 28974/66619 (43.5%)</li><li>Warnings: 0/66619 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Timeout or crash</li>

</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 38643/66619 (58.0%)</li>
<li>Failed: 0/66619 (0.0%)</li>
<li>Not Supported: 27976/66619 (42.0%)</li><li>Warnings: 0/66619 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 41269/66619 (61.9%)</li>
<li>Failed: 0/66619 (0.0%)</li>
<li>Not Supported: 25350/66619 (38.1%)</li><li>Warnings: 0/66619 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by jayfoad** [comment]
Ping for re-review please.

**Comment by Flakebi** [comment]
Could you rebase your branch, to see if the tests are passing now? (as the CI is currently failing)

**Comment by Flakebi** [comment]
The tests still seem to fail (probably because our llvm has not been updated on github).

**Comment by jayfoad** [comment]
retest this please

**Comment by Flakebi** [comment]
FYI, the retest command only triggers jenkins. For the GitHub CI, you need to rebase or press the re-run button in the ui.
I think the github llvm is only updated when Tim or Guyu does a promotion, which I think was skipped last week, so our llvm does not yet include the upstream change.

**Comment by trenouf** [comment]
Is this on hold until we update llvm, and then it magically needs to be merged at exactly the same time?

**Comment by Flakebi** [comment]
The tests were disabled in a previous, already merged PR, so there’s no need to sync with an llvm update. This PR updates and re-enables the tests, so it can go in any time after llvm is updated.

**Review by Flakebi — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit 91c6233170c031d5837327bc06cd096a01b64a28
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 39f8940199e60c44d4211cf8165dfd12876316fa</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev b638e90ca4e6e5a6fc4f00029d62f8e064aa18eb</li>
<li>SPVGEN: dev 6c2f36bcf0f8ff7ba8d20c8b789ca6346d1e16b1</li>
<li>XGL: dev 89a4cf115b9a2972ff00e2fe4959f4f1c8e7175b</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev 8ad081ccb29a0e35a0ed6099d4e23b02a82e3c2d</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/199929)</summary><ul><li>Built with version 1.3.0.0</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 37585/66643 (56.4%)</li>
<li>Failed: 0/66643 (0.0%)</li>
<li>Not Supported: 29058/66643 (43.6%)</li><li>Warnings: 0/66643 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 37436/66643 (56.2%)</li>
<li>Failed: 0/66643 (0.0%)</li>
<li>Not Supported: 29207/66643 (43.8%)</li><li>Warnings: 0/66643 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 38645/66643 (58.0%)</li>
<li>Failed: 0/66643 (0.0%)</li>
<li>Not Supported: 27998/66643 (42.0%)</li><li>Warnings: 0/66643 (0.0%)</li>
</ul></details>


</ul></details>

**Comment by jayfoad** [comment]
Ping for review please. (I don't understand why @Flakebi's approval does not qualify -- are they not an llpc approver?)

**Comment by Flakebi** [comment]
I’m not a qualified approver, see the [member list](https://github.com/orgs/GPUOpen-Drivers/teams/llpcapprovers/members).
So, @trenouf or @amdrexu need to approve.

**Review by trenouf — APPROVED** [review]
LGTM

**Pull request merged**
